src/compas/datastructures/cell_network/cell_network.py

```diff
@@ -728,6 +728,32 @@ def add_cell(self, faces, ckey=None, attr_dict=None, **kwattr):
 
         return ckey
 
+
+    def add_mesh(self, input_mesh: Mesh):
```

Comment:
i have the felling this already exists but under a different name, no @romanarust ?

Reply:
no, this does not yet exist